### PR TITLE
Support for continuous rendering

### PR DESCRIPTION
**Pull request by jsanda**
I have added a new function, `ContinuousRender`, which takes a channel as input.
The function appends rows as it receives them from the channel. It then checks
the column widths to see if headers need to be redrawn. 

This may address #46. I have not added any tests yet because I would first like to get some feedback on my approach.

Here is a sample program that uses the new function:

```go
package main

import "github.com/jsanda/tablewriter"
import "os"
import "time"

func main() {
        data := [][]string{
                []string{"A", "The Good", "500"},
                []string{"B", "The Very very Bad Man", "288"},
                []string{"C", "The Ugly", "120"},
                []string{"D", "The Gopher", "800"},
                []string{"E", "The Hound", "800"},
                []string{"F", "A very, very long title", "800"},
                []string{"G", "A somehwat long title", "800"},
                []string{"H", "A short title", "800"},
        }
        rows := make(chan []string)
        //done := make(chan struct{})


        table := tablewriter.NewWriter(os.Stdout)
        table.SetHeader([]string{"Name", "Sign", "Rating"})
        table.SetBorders(tablewriter.Border{Left: true, Top: false, Right: true, Bottom: false})
        table.SetHeaderColor(
            tablewriter.Colors{tablewriter.FgBlueColor},
            tablewriter.Colors{tablewriter.FgBlueColor},
            tablewriter.Colors{tablewriter.FgRedColor})

        go func() {
                table.ContinuousRender(rows)
        }()

        for _, r := range data {
                rows <- r
                time.Sleep(700 * time.Millisecond)
        }
}
```

And the output looks like:

```
$ go run main.go
| NAME |   SIGN   | RATING |
+------+----------+--------+
| A    | The Good |    500 |
| NAME |         SIGN          | RATING |
+------+-----------------------+--------+
| B    | The Very very Bad Man |    288 |
| C    | The Ugly              |    120 |
| D    | The Gopher            |    800 |
| E    | The Hound             |    800 |
| NAME |          SIGN           | RATING |
+------+-------------------------+--------+
| F    | A very, very long title |    800 |
| G    | A somehwat long title   |    800 |
| H    | A short title           |    800 |
```

### CODE REVIEW

**Comment by olekukonko** [comment]
@jsanda thanks for the contribution, can you add some test and update the module path accordingly

**Comment by olekukonko** [comment]
@mattn can I have your opinion on this, please .

File: go.mod [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module github.com/olekukonko/tablewriter
+module github.com/jsanda/tablewriter
```
Comment (mattn) [inline go.mod:1]: Do not change here

**Comment by mattn** [comment]
This is possibly useful.

File: go.mod (hunk 1 above)
Comment (olekukonko) [reply go.mod:1]: I totally agree

**Comment by olekukonko** [comment]
@mattn you never disappoint .... thanks for the support and everything

**Comment by cdoern** [comment]
can this be merged?

**Comment by Maximilan4** [comment]
Is it possible to force merge?)

**Comment by sillydong** [comment]
I think this is an amazing feature, thanks for the hard work and I'm actually looking forward to use it. Do we still have any issue with this PR? Can we merge it?

**Pull request closed without merging**